01-Login/exec.ps1

```diff
@@ -0,0 +1,2 @@
+docker build -t auth0-spring-mvc-webapp .
```

Comment:
Would be nice to tag them under a "samples" category with a "latest" version tag. I was using the next one:

`auth0-samples/{REPO}-{PATH}:latest` -> `auth0-samples/spring-mvc-00login:latest`